Mathlib/Computability/NFA.lean

```diff
@@ -1,7 +1,7 @@
 /-
-Copyright (c) 2020 Fox Thomson. All rights reserved.
+Copyright (c) 2020 Fox Thomson, 2022 Russell Emerine, 2024 Tom Kranz. All rights reserved.
```

Comment:
In mathlib, we just keep the year of the first contribution in the copyright header --- please revert this one.